docs/source/tensors.rst

```diff
@@ -12,29 +12,32 @@ a single data type.
+======================================= ===========================================
+32-bit floating point                   ``torch.float32`` or ``torch.float``
+64-bit floating point                   ``torch.float64`` or ``torch.double``
+16-bit floating point [1]_              ``torch.float16`` or ``torch.half``
+16-bit floating point [2]_              ``torch.bfloat16``
 32-bit complex                          ``torch.complex32`` or ``torch.chalf``
 64-bit complex                          ``torch.complex64`` or ``torch.cfloat``
 128-bit complex                         ``torch.complex128`` or ``torch.cdouble``
-8-bit integer (unsigned)                ``torch.uint8``                             :class:`torch.ByteTensor`     :class:`torch.cuda.ByteTensor`
-8-bit integer (signed)                  ``torch.int8``                              :class:`torch.CharTensor`     :class:`torch.cuda.CharTensor`
-16-bit integer (signed)                 ``torch.int16`` or ``torch.short``          :class:`torch.ShortTensor`    :class:`torch.cuda.ShortTensor`
-32-bit integer (signed)                 ``torch.int32`` or ``torch.int``            :class:`torch.IntTensor`      :class:`torch.cuda.IntTensor`
-64-bit integer (signed)                 ``torch.int64`` or ``torch.long``           :class:`torch.LongTensor`     :class:`torch.cuda.LongTensor`
-Boolean                                 ``torch.bool``                              :class:`torch.BoolTensor`     :class:`torch.cuda.BoolTensor`
-quantized 8-bit integer (unsigned)      ``torch.quint8``                            :class:`torch.ByteTensor`     /
-quantized 8-bit integer (signed)        ``torch.qint8``                             :class:`torch.CharTensor`     /
-quantized 32-bit integer (signed)       ``torch.qint32``                            :class:`torch.IntTensor`      /
-quantized 4-bit integer (unsigned) [3]_ ``torch.quint4x2``                          :class:`torch.ByteTensor`     /
-======================================= =========================================== ============================= ================================
+8-bit integer (unsigned)                ``torch.uint8``
+16-bit integer (unsigned)               ``torch.uint16`` (limited support) [4]_
+32-bit integer (unsigned)               ``torch.uint32`` (limited support) [4]_
+64-bit integer (unsigned)               ``torch.uint64`` (limited support) [4]_
+8-bit integer (signed)                  ``torch.int8``
+16-bit integer (signed)                 ``torch.int16`` or ``torch.short``
+32-bit integer (signed)                 ``torch.int32`` or ``torch.int``
+64-bit integer (signed)                 ``torch.int64`` or ``torch.long``
+Boolean                                 ``torch.bool``
+quantized 8-bit integer (unsigned)      ``torch.quint8``
+quantized 8-bit integer (signed)        ``torch.qint8``
+quantized 32-bit integer (signed)       ``torch.qint32``
+quantized 4-bit integer (unsigned) [3]_ ``torch.quint4x2``
```

Comment:
I know it might a a bit too late, but why not mention float8 types there?

Reply:
@albanD was complaining to me about how they were undocumented lol

Reply:
when are we planning to remove the docs for these qdtypes? these no longer align with our long term goals

Reply:
We probably shouldn't remove it, but I guess we could shunt them to their own section